tests/bsim/bluetooth/audio/src/cap_acceptor_test.c

```diff
@@ -895,10 +1022,51 @@ static void test_cap_acceptor_broadcast(void)
 	for (size_t i = 0U; i < stream_count; i++) {
 		k_sem_take(&sem_broadcast_stopped, K_FOREVER);
 	}
+}
+
+static void test_cap_acceptor_broadcast(void)
+{
+	static struct bt_bap_stream *bap_streams[ARRAY_SIZE(broadcast_sink_streams)];
+	size_t stream_count;
+
+	init();
+
+	pa_sync_to_broadcaster();
+
+	bap_sink_stream(bap_streams, &stream_count);
+
+	sink_wait_for_data();
+
+	wait_for_streams_stop(stream_count);
 
 	PASS("CAP acceptor broadcast passed\n");
 }
 
+static void test_cap_acceptor_broadcast_reception(void)
+{
+	static struct bt_bap_stream *bap_streams[ARRAY_SIZE(broadcast_sink_streams)];
+	size_t stream_count;
+
+	init();
+
+	WAIT_FOR_FLAG(flag_pa_request);
+
+	pa_sync_create();
+
+	bap_sink_stream(bap_streams, &stream_count);
+
+	sink_wait_for_data();
+
+	base_wait_for_metadata_update();
+
+	backchannel_sync_send_all(); /* let broadcaster know we can stop the source
```

Comment:
we should remove the metadata_update check here ,so that the  test is identical as the acceptor_broadcast test